### PR TITLE
Expereince card section

### LINKED ISSUE #80: Experience Page
( Try to have cards depicting a picture or graphics related to the workplace + Role Name + Role Time + Bullet points about the work done + A micro-animation for an icon redirecting to any letter if available)
- [ ] Internships/Jobs
- [ ] Mentorship
- [ ] Freelance/Volunteering

** Please feel free to use your own academic data or use my or @anushbhatia 's.

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Fixes #80 70

# Work Done:

- Created the Reaac.js version of  experience  Page
- updated the images
- included actual experience data
-  mapped over the array of data.
 ## Question for the reviewer
- I don't understand why the vercel is showing me an error 404 when I click on on the deployment error

Screenshot 2021-07-11 at 08 10 23

### CODE REVIEW

File: src/pages/Experience/Experience.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
+.my_experience {
+  color: #4e00bb;
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.css:2]: Please define variables for this

File: src/pages/Experience/Experience.js [hunk 2]
```diff
@@ -1,11 +1,93 @@
...
+      title: "Software Development Intern",
+      cardImage: Gnome,
+      place: "Gnome",
+      time: "(May, 2020 - present)",
+      desp: [
+        {
+          text: " Created a modern new interface that contributed to a 15% increase in new user adoption rates within one month of rollout",
+        },
+        {
+          text: "Contributed to rollout of upgraded DEFSafe platform, delivering the company’s first persistent, end-to-end data protection that allow users to control security protocols across multiple devices",
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:20]: A single line should work.

File: src/pages/Experience/Experience.js [hunk 3]
```diff
@@ -1,11 +1,93 @@
-import React from 'react';
+import React, { useState } from "react";
+import styles from "./Experience.css";
+import Gnome from "./../../assets/images/workex/gnome.svg";
+import Layer5 from "../../assets/images/workex/layer5-white.png";
+import Google from "./../../assets/images/workex/google.png";
+// AOS.init();
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:6]: This is not required for now

File: src/pages/Experience/Experience.js [hunk 4]
```diff
@@ -1,11 +1,93 @@
...
-    return (
+  const exp = [
+    {
+      title: "Software Development Intern",
+      cardImage: Gnome,
+      place: "Gnome",
+      time: "(May, 2020 - present)",
+      desp: [
+        {
+          text: " Created a modern new interface that contributed to a 15% increase in new user adoption rates within one month of rollout",
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:17]: Single text input should work.

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Review by anushbhatia — COMMENTED** [review]
Avoid all the warnings before deploying also to import images see the relevant standards in react and you will be able to deploy it successfully.

**Comment by Anita-ihuman** [comment]
@anushbhatia I don't know why the Vercel deployment is failing. Can you help me, please

File: src/pages/Experience/Experience.css [hunk 5]
```diff
@@ -0,0 +1,167 @@
+.card1 {
+  background: white;
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.css:2]: Define the colors for the CSS in variables please

File: src/pages/Experience/Experience.js [hunk 6]
```diff
@@ -1,11 +1,78 @@
...
+              </article>
+            </div>
+          ))}
         </div>
-    );
-}
+      </section>
+    </>
+  );
+};
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:76]: Check if the last semi-colon leads to any warning please

**Comment by anushbhatia** [comment]
> @anushbhatia I don't know why the Vercel deployment is failing. Can you help me, please

Check the warnings present in the vs code's 
image
The sample image is attached below here instead of the red icons there might be yellow warnings symbols to see how to resolve them and then you will be able to resolve it. Or contact me on slack or discord regarding this will help you with it.

File: src/pages/Experience/Experience.css [hunk 7]
```diff
@@ -0,0 +1,167 @@
+.card1 {
+  background: rgb(255, 255, 255);
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.css:2]: use variables for defining colors. Check out the master branch code

File: src/pages/Experience/Experience.css [hunk 8]
```diff
@@ -0,0 +1,167 @@
+.card1 {
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.css:1]: name these classes appropriately

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Responsivenss_Bug
Multiple Responsiveness Bug

**Pull request closed without merging**